svn-folder-upgrade/provision.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+echo " * Running all PHP provisioners"
```

Comment:
Does this need to be here?

Reply:
```suggestion
echo " * Upgrading SVN to Git in progress"
```

Reply:
This doesn't migrate  svn to git, it upgrades SVN repos from the old format to  the new repo format, via `svn upgrade`.

I thought I had changed this but seems I missed it, good catch